docs/01-app/02-guides/prefetching.mdx

```diff
@@ -0,0 +1,348 @@
+---
+title: Prefetching
+description: Learn how to configure prefetching in Next.js
+---
+
+Prefetching makes navigating between different routes in your application feel instant. Next.js tries to intelligently prefetch by default, based on the links used in your application code.
+
+This guide will explain how prefetching works and show common implementation patterns:
+
+- [Automatic prefetch](#automatic-prefetch)
+- [Manual prefetch](#manual-prefetch)
+- [Hover-triggered prefetch](#hover-triggered-prefetch)
+- [Extending or ejecting link](#extending-or-ejecting-link)
+- [Disabled prefetch](#disabled-prefetch)
+
+## How does prefetching work?
+
+When navigating between routes, the browser requests assets for the page like HTML and JavaScript files. Prefetching is the process of fetching these resources _ahead_ of time, before you navigate to a new route.
+
+Next.js automatically splits your application into smaller JavaScript chunks based on routes. Instead of loading all the code upfront like traditional SPAs, only the code needed for the current route is loaded. This reduces the initial load time while other parts of the app are loaded in the background. By the time you click the link, the resources for the new route have already been loaded into the browser cache.
+
+When navigating to the new page, there's no full page reload or browser loading spinner. Instead, Next.js performs a [client-side transition](/docs/app/getting-started/linking-and-navigating#client-side-transitions), making the page navigation feel instant.
+
+## Prefetching static vs. dynamic routes
+
+|                                                                      | **Static page** | **Dynamic page**                             |
+| -------------------------------------------------------------------- | --------------- | -------------------------------------------- |
+| **Prefetched**                                                       | Full Route      | Skipped or shared layouts and loading states |
+| [**Client Cache TTL**](/docs/app/deep-dive/caching#full-route-cache) | 5 min (default) | Off unless enabled                           |
+| **Server roundtrip on click**                                        | No              | Yes (streamed after shell)                   |
```

Comment:
For streamed after shell, link to "Partial Prerendering (PPR)" below in this file?